src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoDatabaseCustomNode.cs

```diff
@@ -32,6 +32,9 @@ protected override void InitializeChild(TreeNode parent, TreeNode child, object
 
         public override string GetNodeSubType(object smoObject, SmoQueryContext smoContext)
         {
+            // IsLedger property check is disabled on Azure databases to avoid making connections to databases 
+            // individually when listing databases in Object Explorer since it's value is not available from 'master' DB for all databases..
```

Comment:
I would suggest adding this as a comment in the XML instead - that's where this is actually being done and someone could easily later on go change that back to All and not realize what they're doing.